examples/fabric-sync/bridge/src/BridgedDeviceManager.cpp

```diff
@@ -250,7 +250,6 @@ int BridgedDeviceManager::RemoveDeviceEndpoint(BridgedDevice * dev)
     {
         if (mDevices[index].get() == dev)
         {
```

Comment:
could we add a `assertChipStackLockedByCurrentThread` on top of these methods to ensure we do not flip from too much locking to too little?

Reply:
Done